test/CMakeLists.txt

```diff
@@ -24,3 +24,9 @@ if(TARGET MQT::LogicBlocks)
   package_add_test(mqt-logic-blocks-test MQT::LogicBlocks
                    ${CMAKE_CURRENT_SOURCE_DIR}/test_logicblocks.cpp)
 endif()
+
+if(TARGET MQT::QMapHybrid)
+  file(COPY ${PROJECT_SOURCE_DIR}/test/hybridmap DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Changed that, hopefully it works.

Reply:
It did not work, although I'm not sure why. On my local machine the new version works fine but the CI complains it can't find the files so I reverted to the original version. If you know what the problem is, let me know.

Reply:
I'll see whether I can resolve that locally. It's probably about the working directory when running the tests. These things are really hard to get right and always make problems. That's why I much prefer circuits to be defined in the C++ or Python sources themselves which completely eliminates these problems.
Same goes for the architectures. Here one could create an `std::stringstream` with the config to avoid the filesystem...
I'll try to have a look once most of the other conversations have been resolved.